pagopa/io-app-design-system
# [SIW-1294] Update `ModuelCredential` component

## Short description
This PR updates the `ModuleCredential` component allowing to show a loading spinner instead of the `chevronRight` icon. It also introduces two new icons, `Accessibility` and `Car`.

## List of changes proposed in this pull request
- Updated `ModuleCredential` with `isFetching` prop
- Updated example app with a new example for `ModuleCredential` component
- Added `Accessibility` and `Car` icons

## How to test
Run the example app, go to **Modules** section and check that the `ModuleCredential` component is rendered correclty

## Preview

https://github.com/pagopa/io-app-design-system/assets/6160324/93c1ecfd-0a1a-4151-8774-d3401c8ee1f9
